compiler/rustc_interface/src/passes.rs

```diff
@@ -608,6 +617,8 @@ pub static DEFAULT_QUERY_PROVIDERS: LazyLock<Providers> = LazyLock::new(|| {
     providers.analysis = analysis;
     providers.hir_crate = rustc_ast_lowering::lower_to_hir;
     providers.resolver_for_lowering = resolver_for_lowering;
+    providers.stripped_cfg_items = stripped_cfg_items;
+    providers.resolutions = resolutions;
```

Comment:
`stripped_cfg_items` and `resolutions` are trivial and used once, they can be turned into closures here.